src/main/java/org/spongepowered/api/data/property/LocationBasePropertyHolder.java

```diff
@@ -25,36 +25,38 @@
 package org.spongepowered.api.data.property;
 
 import com.flowpowered.math.vector.Vector3i;
-import org.spongepowered.api.data.Property;
 import org.spongepowered.api.util.Direction;
 
 import java.util.Collection;
+import java.util.Map;
 import java.util.Optional;
+import java.util.OptionalDouble;
+import java.util.OptionalInt;
 
 public interface LocationBasePropertyHolder {
 
     /**
-     * Attempts to retrieve a specific {@link Property} type of the block at the
-     * specific location. If the property is not applicable,
+     * Attempts to retrieve a value for the specified {@link Property} from
+     * the block at the specified location. If the property is not applicable,
      * {@link Optional#empty()} is returned.
      *
      * <p>{@link Property}s can define various immutable information about a
      * {@link PropertyHolder} that is dependent on the instance of the holder.
      * As {@link Property}s cannot be changed, the {@link PropertyHolder} can
      * not change the information about it's own properties either.</p>
      *
```

Comment:
May help to add a link to 
```
{@see Properties}
```
 to aid developers to look at what possible properties exist